nvidia.go

```diff
@@ -0,0 +1,116 @@
+//
```

Comment:
not sure, but I think this header should have a copyright, maybe  `Copyright (c) 2020 Aohan Young` .. ?

cc @grahamwhaley

Reply:
Not sure whether it's necessary.Maybe later to add.

Reply:
@devimc is correct - please add a copyright comment to all new files.

Related: I've just created https://github.com/kata-containers/tests/issues/2323.

/cc @grahamwhaley